test/tests/ide/jetbrains/warmup-indexing.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+
+
+set -euo pipefail
```

Comment:
Addressed

```
# This script is used to test JetBrains prebuild warmup indexing (search warmup-indexing.sh in codebase)
# It will get the last indexing json file (scan type FULL_ON_PROJECT_OPEN)
# and check if the scheduled indexing count is 0
#
# `exit 0` means JetBrains IDEs no need to indexing again
```